source/common/common/utility.h

```diff
@@ -501,11 +501,12 @@ template <typename Value> class IntervalSetImpl : public IntervalSet<Value> {
 };
 
 /**
- * Hashing functor for use with unordered_map and unordered_set with string_view as a key.
+ * Hashing functor for use with absl hash tables, which can use hash on
+ * string_view if this declaration is visible.
  */
-struct StringViewHash {
-  std::size_t operator()(const absl::string_view& k) const { return HashUtil::xxHash64(k); }
-};
+template <typename H> H AbslHashValue(H h, absl::string_view str) {
```

Comment:
Is this necessary? `absl::Hash` intrinsically supports `absl::string_view`.

Reply:
more than not necessary, this is forbidden for users to do

Reply:
that makes sense; I don't own string_view so I shouldn't define a global hash policy for it.